ui/src/dashboards/components/rename_dashboard/RenameDashboard.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {Component, KeyboardEvent} from 'react'
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import './RenameDashboard.scss'
+import {
+  DASHBOARD_NAME_MAX_LENGTH,
+  DEFAULT_DASHBOARD_NAME,
+} from 'src/dashboards/constants/index'
+
+interface Props {
+  onRename: (name: string) => void
+  name: string
+}
+
+interface State {
+  isEditing: boolean
+  reset: boolean
+}
+@ErrorHandling
+class DashboardEditHeader extends Component<Props, State> {
```

Comment:
Should this be called `RenameDashboard`?

Reply:
Good catch!